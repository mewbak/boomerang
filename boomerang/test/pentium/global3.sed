1{
i\
void foo1();\
void foo2();\
int local5;
}
