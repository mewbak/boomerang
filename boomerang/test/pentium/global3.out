long long a = 123456789012345;
int b = 7;

int main(int param1)
{
    foo1(param1 - 16);
    printf("b = %i\n", b);
    return 0;
}

void foo1(int param5)
{
    foo2();
    return;
}

void foo2()
{
    b = 12;
    local5 = printf("a = %lld\n", a);
    return;
}

